dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -112,6 +136,12 @@ services:
       options:
         syslog-address: "tcp://localhost:5140"
         syslog-facility: "local3"
+<% if scope.lookupvar('compose::codenvy_extra_hosts') != 'NULL' -%>
+    extra_hosts:
+<% scope.lookupvar('compose::codenvy_extra_hosts').split(",").each do |val| -%>
+<%='      - "' + val + '"' %>
+<% end -%>
+<% end -%>
```

Comment:
if it's possible also to use some sort of macros to avoid copy/paste of the same code on each container